torch/csrc/utils/python_symnode.h

```diff
@@ -100,6 +101,12 @@ class PythonSymNodeImpl : public c10::SymNodeImpl {
     return getPyObj().attr("is_nested_int")().is(py::handle(Py_True));
   }
 
+  c10::TensorImpl* nested_int_vec() const override {
+    TORCH_INTERNAL_ASSERT(
+        false,
+        "Expect not to call nested_int_vec on PythonSymNodeImpl in C++");
+  }
```

Comment:
`class SymNode` has a method `nested_int_vec`. Usually methods on python_symnode.h would redirect there, but since the factory function creation logic is the only thing today that calls nested_int_vec and that is done in python from torch dispatch, there is no need to support this in cpp today.